src/main/cpp/subsystems/Flywheel.cpp

```diff
@@ -84,14 +87,17 @@ bool Flywheel::IsShooting() const {
 void Flywheel::Reset() {
     m_controller.Reset();
     m_encoder.Reset();
+    m_lastAngle = GetAngle();
```

Comment:
m_angle should be initialized as well.

Reply:
If m_angle is initialized at the top of controller periodic why does it need to be initialized there as well?

Reply:
If someone decides to call GetAngularVelocity() without starting the controller first, you'll get garbage for the measurement.